builtins/amp-video.js

```diff
@@ -36,11 +36,31 @@ export function installVideo(win) {
     }
 
     /** @override */
-    layoutCallback() {
+    buildCallback() {
+      /** @private @const {?HTMLVideoElement} */
```

Comment:
I don't think it should be, I just kept the same annotation type previously used, will update this to be not nullable (that's {!...} right?)